<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Алгоритмы</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_step"/>
    <addaction name="actionDDA"/>
    <addaction name="action_bres"/>
    <addaction name="action_circle"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_step">
   <property name="text">
    <string>Пошаговый</string>
   </property>
  </action>
  <action name="actionDDA">
   <property name="text">
    <string>DDA-линия</string>
   </property>
  </action>
  <action name="action_bres">
   <property name="text">
    <string>Брезенхем</string>
   </property>
  </action>
  <action name="action_circle">
   <property name="text">
    <string>Брезенхем (окружность)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
